server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js

```diff
@@ -15,18 +15,20 @@ const DataCol = require('../schemaRdb/dataCol')
 
 const NodeRepository = require('../../../record/persistence/nodeRepository')
 
-const runSelect = async (surveyId, tableName, cols, offset, limit, filterExpr, sort = '', client) => {
+const runSelect = async (surveyId, tableName, cols, offset, limit, filterExpr, filterStr, sort = '', client) => {
```

Comment:
I don't like having two different parameters that do the same thing. Please use only one

Reply:
OK, done